libs/sdk-py/langgraph_sdk/client.py

```diff
@@ -239,15 +239,57 @@ def __init__(self, http: HttpClient) -> None:
         self.http = http
 
     async def get(self, assistant_id: str) -> Assistant:
-        """Get an assistant by ID."""
+        """Get an assistant by ID.
+
+        Args:
+            assistant_id (str): The ID of the assistant to get.
+
+        Returns:
+            Assistant: The returned assistant.
+
+        Example Usage:
```

Comment:
Could you check that this renders nicely? I think mkdocstrings checks for `Example:` and is very sensitive to spacing